### DISCUSSION TITLE
Bresser-5in1 indoor receiving problem

### DISCUSSION THREAD

**Discussion by marcomiani** [merbanan/rtl_433#2153] [General]
Hi, I have a Bresser-5in1 weather station  whose signal is correctly received and decoded, with 119 protocol, when I put the USB  RTL dongle antenna ( a Fitipower FC0012 tuner ) outside the door of my house ( RSSI about -10dB ). If I move the antenna just 2meters inside the house no data is received even if the  internal unit of the weather station correctly receive data.  I have no clue about what to try, can someone give me any hint? thank you very much

**Comment by zuckschwerdt** [comment]
The `-Y autolevel` option is recommended to extend range.

**Reply by marcomiani** [reply]
thank you for your reply. 
this is the command line
`rtl_433 -f 868.28M -s 1M  -R 119  -Y autolevel`
just outside the door I receive decoded data, 2 meters inside nothing

**Comment by zuckschwerdt** [comment]
Also use `-M level -M noise` to check the levels and what the autolevel is doing.

**Comment by marcomiani** [comment]
outside
Estimated noise level is -32.3 dB, adjusting minimum detecton level to -29.3 dB
Current signal level -14.2 dB, estimated noise -31.8 dB
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
time      : 2022-08-18 16:15:20
model     : Bresser-5in1 id        : 15
Battery   : 1            Temperature: 25.0 C       Humidity  : 68            Wind Gust : 2.0 m/s
Wind Speed: 0.8 m/s      Direction : 112.5         Rain      : 6.4 mm        Integrity : CHECKSUM
Modulation: FSK          Freq1     : 868.2 MHz     Freq2     : 868.4 MHz
RSSI      : -8.7 dB      SNR       : 24.5 dB       Noise     : -33.1 dB
Current noise level -32.4 dB, estimated noise -32.4 dB
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
time      : 2022-08-18 16:15:32
model     : Bresser-5in1 id        : 15
Battery   : 1            Temperature: 25.1 C       Humidity  : 67            Wind Gust : 2.0 m/s
Wind Speed: 0.9 m/s      Direction : 112.5         Rain      : 6.4 mm        Integrity : CHECKSUM
Modulation: FSK          Freq1     : 868.2 MHz     Freq2     : 868.3 MHz
RSSI      : -8.5 dB      SNR       : 28.9 dB       Noise     : -37.4 dB
Current noise level -32.2 dB, estimated noise -32.3 dB
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
time      : 2022-08-18 16:15:44
model     : Bresser-5in1 id        : 15
Battery   : 1            Temperature: 25.1 C       Humidity  : 67            Wind Gust : 2.0 m/s
Wind Speed: 0.9 m/s      Direction : 112.5         Rain      : 6.4 mm        Integrity : CHECKSUM
Modulation: FSK          Freq1     : 868.2 MHz     Freq2     : 868.3 MHz
RSSI      : -12.4 dB     SNR       : 26.7 dB       Noise     : -39.1 dB

inside
Estimated noise level is -30.3 dB, adjusting minimum detection level to -27.3 dB
Current noise level -30.0 dB, estimated noise -30.1 dB
Current noise level -30.0 dB, estimated noise -30.1 dB
Current noise level -30.1 dB, estimated noise -30.1 dB
Current noise level -30.2 dB, estimated noise -30.1 dB
Current noise level -30.1 dB, estimated noise -30.1 dB
Current noise level -30.2 dB, estimated noise -30.1 dB

**Reply by merbanan** [reply]
Try with -g values from 0-20. Maybe some value will work better.

**Reply by marcomiani** [reply]
tried all the values,  the tuner gain was set to  7.1, 17.9, 19.2 dB , no data decoded

**Comment by zuckschwerdt** [comment]
That's a clear band with really no noise at all. The AGC of RTL-SDr is too low/slow then. IIRC the Fitipower FC0012 is known for that. Perhaps try a fixed gain (e.g. `-g 20`).

**Reply by marcomiani** [reply]
no results even with -g 20  
Can you confirm that if it's the dongle fault  I only have two options:
- better dongle (  can you suggest me something ? )
- move current receiver near the weather station 

thank you very much for your support

**Comment by zuckschwerdt** [comment]
19.2 dB is the maximum gain :/
https://github.com/osmocom/rtl-sdr/blob/master/src/tuner_fc0012.c#L336

You could switch from (Osmocom, default) rtlsdr to [librtlsdr](https://github.com/librtlsdr/librtlsdr/) -- more recent and optimized.

Other people might known what to do about the Fitipower FC0012, I don't remember, sorry.

The popular general purpose RTL-SDR is the R820T/R820T2.
